src/mbgl/renderer/image_manager_observer.hpp

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
+#include <mbgl/renderer/renderer_observer.hpp>
+
 namespace mbgl {
 
 class ImageManagerObserver {
 public:
     virtual ~ImageManagerObserver() = default;
 
-    virtual void onStyleImageMissing(const std::string&, std::function<void()>) {}
+    using StyleImageMissingCallback = std::function<void()>;
```

Comment:
nit: `std::function<void()>` looks short enough to be kept, also it brings more readability on the semantics than `StyleImageMissingCallback`

Reply:
Changed in 3637d15a8fc2be6318c33c22d9cf8b5ecbf00737